servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1900,6 +1904,11 @@ void RenderForwardClustered::_render_scene(RenderDataRD *p_render_data, const Co
 
 	bool using_sss = rb_data.is_valid() && !is_reflection_probe && scene_state.used_sss && ss_effects->sss_get_quality() != RS::SUB_SURFACE_SCATTERING_QUALITY_DISABLED;
 
+	if (using_sss && p_render_data->transparent_bg) {
+		WARN_PRINT_ONCE("Sub-surface scattering is not supported in viewports with a transparent background. Disabling SSS in transparent viewport.");
+		using_sss = false;
+	}
+
 	if ((using_sss || ce_needs_separate_specular) && !using_separate_specular) {
```

Comment:
This check makes me curious: are *all* compositor effects that need separate specular broken when using a transparent background? If so, we should add a dedicated warning here.